segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorage.java

```diff
@@ -662,14 +634,21 @@ public boolean supportsTruncation() {
 
     @Override
     public void close() {
+        close("metadataStore", this.metadataStore);
+        close("garbageCollector", this.garbageCollector);
+        this.closed.set(true);
+    }
+
+    private void close(String message, AutoCloseable toClose) {
         try {
-            if (null != this.metadataStore) {
-                this.metadataStore.close();
+            log.info("{} Closing {}", logPrefix, message);
+            if (null != toClose) {
+                toClose.close();
             }
+            log.info("{} Closed {}", logPrefix, message);
```

Comment:
Let's not pollute the logs with so many messages. Make the "Closing" one `debug`.

Reply:
done